### ISSUE TITLE
Subtle Change to the Advisor Scroll

### ISSUE THREAD

**Issue by SanctumMK** [OpenThrone/OpenThrone#269]
Hi there, first time using GitHub so thought I'd start with something very subtle and not too meaningful of a change to see if this is the correct way of doing things.

I've added a texture over the current Advisor Scroll to make it look a little more "used" and paper-like.
There's also a second addition of the scroll with the texture and a drop shadow to give it a bit more depth.

Let me know if this is the correct way of putting forward such a change and I can start working on some bigger changes to current art and creating new art entirely.

Nothing I use is AI generated and I would be happy for it to be used freely. :)

**With texture added**
![Advisor Scroll (with texture)](https://github.com/user-attachments/assets/f6a5ca5c-3fe7-4084-93cf-d53cb3ab3d2a)

**With texture and drop shadow added**
![Advisor Scroll (with texture and shadow)](https://github.com/user-attachments/assets/07271859-83fd-43b7-b6bf-6d2f7b546af9)

**Comment by uaktags** [comment]
This looks good, the drop-shadow should be able to be done via css, but i'll give it some play to see. The original source of the scroll is a pretty jank illustrator vector that i've just rasterized.

**Comment by uaktags** [comment]
So my personal hope is to get more icons created for the UI/UX, before starting to really focus on Game Design elements (like characters, items, etc). All that i think will come later and is low priority. 

For icons, we use a mix of [FontAwesome](https://fontawesome.com/) and [RPG-Awesome](https://nagoshiashumari.github.io/Rpg-Awesome/) 
RPG Awesome is pretty great and with someone who can design the font icons, I can easily convert those into the necessary package to get them usable on the site. 

Outside of icons, textures, much like how you did with the scroll would also be great. The artist i commissioned for the scroll did try to make some textures to be used for things like the nav bar, but ultimately just didn't enjoy the outcome which is why we're just plan colors right now.

**Comment by SanctumMK** [comment]
> So my personal hope is to get more icons created for the UI/UX, before starting to really focus on Game Design elements (like characters, items, etc). All that i think will come later and is low priority. 
> 
> For icons, we use a mix of [FontAwesome](https://fontawesome.com/) and [RPG-Awesome](https://nagoshiashumari.github.io/Rpg-Awesome/) 
> RPG Awesome is pretty great and with someone who can design the font icons, I can easily convert those into the necessary package to get them usable on the site. 
> 
> Outside of icons, textures, much like how you did with the scroll would also be great. The artist i commissioned for the scroll did try to make some textures to be used for things like the nav bar, but ultimately just didn't enjoy the outcome which is why we're just plan colors right now.

Sounds good to me!

Can you give me a list of icons you're looking for? I can attempt to crack on with those ASAP. 

I can also look at bringing some texture back to the main banners.

Let me know, always happy to help!

**Comment by uaktags** [comment]
Closing this in favor of the updated version. Way to go, they are all great!
